packages/frameworks/src/frameworks.ts

```diff
@@ -1960,6 +1960,49 @@ export const frameworks = [
       },
     ],
   },
+  {
+    name: 'FastHTML (Experimental)',
+    slug: 'fasthtml',
+    demo: 'https://fasthtml-template.vercel.app',
+    logo: 'https://api-frameworks.vercel.sh/framework-logos/fasthtml.svg',
+    tagline: 'The fastest way to create an HTML app',
+    description:
+      'A library for writing fast and scalable Starlette-powered web applications',
+    website: 'https://fastht.ml',
+    useRuntime: { src: 'main.py', use: '@vercel/python' },
+    detectors: {
```

Comment:
Can we also look for `python-fasthtml` inside `requirements.txt`? I think we should be more specific in case a stray `main.py` is in someone's non-fasthtml project.

```
{
  path: 'requirements.txt',
  matchContent:
    'python-fasthtml',
},
```